src/core/loader.h

```diff
@@ -0,0 +1,13 @@
+// SPDX-FileCopyrightText: Copyright 2024 shadPS4 Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <string>
```

Comment:
This should be be placed into the Loader namespace

Reply:
ok